stdlib/public/runtime/RefCount.cpp

```diff
@@ -0,0 +1,137 @@
+//===--- RefCount.cpp -----------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/Runtime/HeapObject.h"
+
+namespace swift {
+
+template <typename RefCountBits>
+void RefCounts<RefCountBits>::incrementSlow(RefCountBits oldbits,
+                                            uint32_t n) {
+  if (oldbits.hasSideTable()) {
+    // Out-of-line slow path.
+    auto side = oldbits.getSideTable();
+    side->incrementStrong(n);
+  }
+  else {
```

Comment:
Why is the else here on another line? (I am asking since below you are writing '} else {'.